frontend/packages/kubevirt-plugin/src/components/form/form-row.tsx

```diff
@@ -39,30 +40,34 @@ export const FormRow: React.FC<FormRowProps> = ({
         type === ValidationErrorType.Info || type === ValidationErrorType.Warn ? message : undefined
       }
       className={className}
+      labelIcon={
+        help && (
+          <span>
+            <Popover
+              position={PopoverPosition.right}
+              aria-label={`${fieldId} help`}
+              onHide={onHide}
+              onShow={onShow}
+              bodyContent={help}
+            >
+              <Button variant="plain" className="pf-c-form__group-label-help">
+                <StatusIconAndText
+                  title={`${fieldId} help`}
+                  iconOnly
+                  icon={
+                    <HelpIcon
+                      className={classnames({
+                        'kubevirt-form-row__help-icon--hidden': !isActive,
+                      })}
+                    />
+                  }
+                />
+              </Button>
+            </Popover>
```

Comment:
Maybe I'm missing something but shouldn't this code for the popover be closer to what PF supplies? It seems very different.
```<Popover
              headerContent={
                <div>
                  The <a href="https://schema.org/name">name</a> of a <a href="https://schema.org/Person">Person</a>
                </div>
              }
              bodyContent={
                <div>
                  Often composed of{' '}
                  <a href="https://schema.org/givenName" target="_blank">
                    givenName
                  </a>{' '}
                  and{' '}
                  <a href="https://schema.org/familyName" target="_blank">
                    familyName
                  </a>
                  .
                </div>
              }
            >
              <button
                aria-label="More info for name field"
                onClick={e => e.preventDefault()}
                aria-describedby="simple-form-name"
                className="pf-c-form__group-label-help"
              >
                <HelpIcon noVerticalAlign />
              </button>
            </Popover>
```

Reply:
I have updated it to align closer to what PF has. I'm not sure why it was originally written to use the `StatusIconAndText` component since it's not providing a status. I also don't know why it was using a `Button` component rather than a button element since it's not being used as a standard button.